src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Wysiwyg.Edit.cshtml

```diff
@@ -20,6 +24,11 @@
 
 <script at="Foot">
     $(function () {
+        @if(cultureAspect.LanguageDirection == LanguageDirection.RTL)
```

Comment:
I think we can just call `$.trumbowyg.langs['en']._dir = 'rtl'` as this is the default embedded language, and this is how the other languages assign it. I believe that at some point we'll want to also include the localization files from trumbowyg. Or leave it to another module with a custom localized version of trumbowyg if there are licensing or other distribution issues.

Reply:
And we don't even have to add an `if`, just assign whatever you have in the direction property.

Reply:
It's good and bad because we should aware about the current language for the trumbowyg, but while it supports `en` by default I can add a comment before this line

Reply:
> I think we can just call $.trumbowyg.langs['en']._dir = 'rtl'

I tried that with no luck :(